cloud/datasync/test/SyncServerTest.ts

```diff
@@ -74,3 +76,42 @@ describe('FeedHenry Sync Tests', function() {
     });
   });
 });
+
+@suite
```

Comment:
As for typemoq it doesn't look like it's actually being used here. I think some of the other modules had success with sinon, I'd rather have it instead just because it's so much, much more popular, which makes it more familiar to community contributors.